### PR TITLE
Matrix-representation graphs in C

### PR DESCRIPTION
**Pull request by abdouskamel**
**Fixes issue:** #3070

**Changes:**
Hello everybody.

Here is an implementation in C of matrix-representation graphs. Hope you enjoy it. :)

### CODE REVIEW

File: code/graph_algorithms/data_structures/adjacency_matrix_C/main.c [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+#include "mgraph_struct.h"
+#include <stdio.h>
+
+#define NB_NODES 5
+
+int
+main()
+{
+  M_Graph g;
```
Comment (arnavb) [inline code/graph_algorithms/data_structures/adjacency_matrix_C/main.c]: I’m not entirely sure on this, but it seems you are using the wrong naming style for everything. Use camel case for variable and function names. Use pascal case for struct names.

Comment (abdouskamel) [reply code/graph_algorithms/data_structures/adjacency_matrix_C/main.c]: When i write it MGraph, it seems too contiguous, so i wrote it M_Graph.

Comment (arnavb) [reply code/graph_algorithms/data_structures/adjacency_matrix_C/main.c]: Ok

Comment (abdouskamel) [reply code/graph_algorithms/data_structures/adjacency_matrix_C/main.c]: Should i edit naming style ?

Comment (AdiChat) [reply code/graph_algorithms/data_structures/adjacency_matrix_C/main.c]: It is fine :+1:

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**